### PR TITLE
Update to use Doximity app

### PR DESCRIPTION
**Pull request by Chandlerdea**
Pivotal story link: https://www.pivotaltracker.com/story/show/171090129

This library was using the old dialer app's URL scheme and app store URL. This changes the code to use the doximity app's URL scheme and app store URL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 ## What is CallWithDoxDialer?
 
-Doximity's [Dialer][] app lets healthcare professionals make phone calls to patients while on the go -- without revealing personal phone numbers. Calls are routed through Doximity's HIPPA-secure platform and relayed to the patient who will see the doctor's office number in the Caller ID. Doximity Dialer is currently available to verified physicians, nurse practitioners, physician assistants and pharmacists in the United States.
+Doximity's lets healthcare professionals make phone calls to patients while on the go -- without revealing personal phone numbers. Calls are routed through Doximity's HIPPA-secure platform and relayed to the patient who will see the doctor's office number in the Caller ID. Doximity is currently available to verified physicians, nurse practitioners, physician assistants and pharmacists in the United States.
```
Comment (bobgilmore) [inline README.md]: Since you lost the "app", "Domimity's" should become "Doximity".

**Review by bobgilmore — CHANGES_REQUESTED** [review]
In addition to the "typo" I flagged in the README.md , you should also bump the Podspec.

**Comment by bobgilmore** [comment]
You might also want to take (or recreate) the other PR that corrects the spelling of "HIPAA." I just approved it.

**Comment by Chandlerdea** [comment]
I'll just correct it here.

**Review by gpambrozio — COMMENTED** [review]
I (still) didn't looked at the whole readme after the change but noticed that we should update this image: https://github.com/doximity/CallWithDoxDialer/blob/master/ReadmeResources/InfoPlistExample.png

File: CallWithDoxDialer.podspec [hunk 2]
```diff
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   Doximity lets healthcare professionals make phone calls 
                   to patients while on the go -- without revealing personal phone numbers.
-                  Calls are routed through Doximity's HIPPA-secure platform and relayed to the patient
+                  Calls are routed through Doximity's HIPAA-secure platform and relayed to the patient
                   who will see the doctor's office number in the Caller ID.
                   Doximity Dialer is currently available to verified physicians, nurse practitioners,
```
Comment (bobgilmore) [inline CallWithDoxDialer.podspec]: The Podspec has a `Doximity Dialer is currently available` reference that should be changed to just `Doximity is currently available` as well...

**Review by bobgilmore — CHANGES_REQUESTED** [review]

**Review by bobgilmore — APPROVED** [review]
Loos good to me!

**Pull request merged**